src/nox_poetry/sessions.py

```diff
@@ -148,6 +181,8 @@ def rewrite(arg: str, extras: Optional[str]) -> str:
         except CommandSkippedError:
             return
 
+        # args += to_global_pip_args(requirements)
+
```

Comment:
This is where I had the config being injected before. I'll update this based on your opinion of how to best proceed.